src/Mono.Android/Java.Interop/JavaConvert.cs

```diff
@@ -57,11 +57,13 @@ static class JavaConvert {
 
 		static Func<IntPtr, JniHandleOwnership, object?>? GetJniHandleConverter (Type? target)
 		{
-			const string justification = "JavaDictionary<,>, JavaList<>, and JavaCollection<> use DynamicallyAccessedMembers for PublicMethods to preserve FromJniHandle().";
-			[UnconditionalSuppressMessage ("Trimming", "IL2055", Justification = justification)]
-			[UnconditionalSuppressMessage ("Trimming", "IL2068", Justification = justification)]
+			[UnconditionalSuppressMessage ("Trimming", "IL2055", Justification = "JavaDictionary<,>, JavaList<>, and JavaCollection<> are preserved by the MarkJavaObjects linker step.")]
```

Comment:
Nice :-)

Nit: The justification doesn't apply to the warning - see my other comment about the reason for the warning.